MovieDB/MovieDB/Scenes/Dashboard/DashboardViewModel.swift

```diff
@@ -2,85 +2,70 @@ import RxCocoa
 import RxSwift
 
 struct DashboardViewModel {
-    let coordinator: DashboardCoordinatorType
     let useCase: DashboardUseCaseType
 }
 
 extension DashboardViewModel: ViewModel {
     struct Input {
         let loadTrigger: Driver<Void>
-        let searchButtonTrigger: Driver<Void>
-        let watchListButtonTrigger: Driver<Void>
         let loadMoreButtonTrigger: Driver<Void>
-        let searchText: Driver<String>
         let categorySelectedTrigger: Driver<CategoryType>
-        let movieSelectedTrigger: Driver<Movie>
+        let pageNumber: BehaviorRelay<Int>
+        let movieItems: BehaviorRelay<[Movie]>
+        let bookmarkButtonTrigger: PublishSubject<Void>
+        let starButtonTrigger: PublishSubject<Void>
     }
     
     struct Output {
-        let movieItems = BehaviorRelay<[Movie]>(value: [])
-        let categoryItems = BehaviorRelay<[CategoryType]>(value: [])
-        let pageNumber = BehaviorRelay<Int>(value: 1)
+        let movieItems: BehaviorRelay<[Movie]>
+        let firstPageMovies: Driver<[Movie]>
```

Comment:
Why declare the firstPageMovies and  nextPageMovies ?
Should only use one let movies: BehaviorRelay<[Movie]> for the current data of this screen
Whenever get a new movies list from any API, check logic and update the movies corresponding.
On viewcontroller, will trigger output and update tableview